doc/operator-notes.md

```diff
@@ -67,7 +67,7 @@ The following table shows the possible combinations of whether or not a partitio
 | false             | true        | true               | Create specified partition
 | true              | false       | false              | Fail
 | true              | false       | true               | Delete existing partition
-| true              | true        | false              | Check if existing partition matches the specified one, fail if it does not
+| true              | true        | false              | Check if existing partition matches the specified one, if it does not match, check if the `resize` field is set to true and all of the specified attributes (`label`, `start`, `uuid`, and `typeGuid`) matches existing partition's attributes except the partition size, then resize an existing partition else fail the execution.
```

Comment:
WDYT about this?

```suggestion
| true              | true        | false              | Check if existing partition matches the specified one, if it does not match, check if the `resize` field is set to true and all of the specified attributes (`label`, `start`, `uuid`, and `typeGuid`) matches existing partition's attributes except the partition size, then resize an existing partition otherwise fail.
```

Reply:
Simpler:
```suggestion
| true              | true        | false              | Verify existing partition matches the specified one, otherwise resize it if `resize` is true and partition matches in all respects except size, otherwise fail
```